templates/vue/cypress/integration/search-bar.spec.js

```diff
@@ -0,0 +1,17 @@
+describe("Search bar", () => {
+  /* beforeEach(() => {
+    cy.fixture("multi-author.json").as("tapestry")
+    cy.setup("@tapestry")
+  }) */
+
+  it("should be able to search nodes by its title", () => {
+    // click search button
+    // assert search by title by default
+    // type in 'hello'
+    // assert correct nodes are visible
```

Comment:
I'm assuming that our "search results" won't be any different than it is now i.e. nodes that match the title in some way will have the regular opacity and nodes that don't will be more transparent.

There's other ways to handle search results, but keeping it the same would be the easiest. Some alternatives:

1. Highlight the parts of the title that match the query (on top of reducing opacity)
2. Show a dropdown of search results that, when an item is clicked, would "pan" the Tapestry to that particular node (this one's a bit of a stretch, but would look nice):

<img width="455" alt="Screen Shot 2020-11-05 at 14 05 57" src="https://user-images.githubusercontent.com/31267630/98301726-1be93a80-1f70-11eb-815e-e3d0018dd98f.png">

Reply:
I think we can keep it the same. There was not any negative feedback regarding the way we show results.

Just keep 2 things in mind here:
1. The selection of search criteria (e.g. "Title" in your mockup) should have an indicator that shows it's a select element (e.g. a down-pointing triangle to its right-handside)
2. If "Status" is selected as the search criteria, we should offer a select element, not auto-complete (default should be "All")

Reply:
Ok - should there be a dropdown as well if we're searching by title?